Runtime/SpriteGroup.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using UnityEditor;
+using UnityEngine;
+using UnityEngine.Serialization;
+
+namespace Unity.MaterialSwitch
+{
+    [ExecuteAlways]
+    [RequireComponent(typeof(SelectionGroups.SelectionGroup))]
+    internal class SpriteGroup : MonoBehaviour
+    {
+        public SpriteRenderer[] spriteRenderers;
```

Comment:
please use `HideInInspector` attribute. We don't want users to touch this in the inspector